domains/cutyoururl.ashishagarwal.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "CutYourURL is a free URL Shortener!",
+  "repo": "https://github.com/ashishagarwal203/CutYourURL",
+  "owner": {
+    "username": "ashishagarwal2023",
+    "twitter": "",
+    "email": "code.with.aasheesh@gmail.com"
+  },
+  "record": {
+    "CNAME": "ashishagarwal.is-a.dev/cutyoururl"
```

Comment:
CNAME cant have / and anything after that (also site gives 404 page)

Reply:
https://ashishagarwal.is-a.dev/CutYourURL/
this was what i meant, but bruh ur response made me struggle...